geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/util/AccumuloSftBuilderTest.scala

```diff
@@ -0,0 +1,54 @@
+/***********************************************************************
+* Copyright (c) 2013-2015 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0 which
+* accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+package org.locationtech.geomesa.accumulo.util
+
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.accumulo.data.DigitSplitter
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.opengis.feature.simple.SimpleFeatureType
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+import scala.collection.JavaConversions._
+
+@RunWith(classOf[JUnitRunner])
+class AccumuloSftBuilderTest extends Specification {
+
+  sequential
+
+  "SpecBuilder" >> {
+    "configure table splitters as strings" >> {
+      val sft1 = new AccumuloSftBuilder()
+        .recordSplitter(classOf[DigitSplitter].getName, Map("fmt" ->"%02d", "min" -> "0", "max" -> "99"))
+        .intType("i")
+        .longType("l")
+        .build("test")
+
```

Comment:
Here's the tricky part with extending builders...I can't do this can I? I think we need some more type magic....

```
      val sft1 = new AccumuloSftBuilder()
        .intType("i")
        .longType("l")
        .recordSplitter(classOf[DigitSplitter].getName, Map("fmt" ->"%02d", "min" -> "0", "max" -> "99"))
        .build("test")
```